### ISSUE TITLE
[prereg/no-gpu] AERA repaired event-memory seed37641 runner after merged #786

### ISSUE THREAD

**Issue by vinceackermann2-sys** [vinceackermann2-sys/tam-research#790]
## Purpose

Preregister the final untouched frozen scientific seed `37641` for the repaired event-aligned AERA capability experiment after the #785/#786 evaluation-materialization correction, while keeping **all scientific execution locked** behind later zero-GPU preauthorization and separate explicit L4 authorization.

This issue authorizes only additive runner/adapter/workflow/protocol/CPU-static engineering needed to make a seed37641 run executable and auditable. It authorizes ZERO Modal scientific execution, ZERO GPU allocation, ZERO scientific model construction/training/evaluation, ZERO checkpoint/result writes, and does not consume seed `37641`.

## Fresh live source boundary

Fresh audit immediately before issue creation:
- `SOURCE_MAIN=5e5d746c70380eb4819534eb5a061a2d142fa26f`
- `SOURCE_TREE=2348be0e21c107d254146a7426b432f6f808687f`
- merged correction PR `#786`, frozen head `90d90d133f40a90a4a1c9c56d49db1880eaa7c4f`
- post-merge CI run `34331135982`, job `102399728992`, attempt `1`, success: `1231 passed, 15 warnings in 103.62s`
- #785 remains the newest AERA successor found; no newer competing AERA capability successor exists.

## Scientific lineage / consumed attempts

- seed `17641`: permanently consumed by #758; scientific capability screen FAIL on the historical pre-repair addressing path. Never reuse or reinterpret as repaired-path evidence.
- seed `27641`: permanently consumed by #783 / run `34325644245` / job `102382150501` / attempt `1`. L4 start marker emitted, but zero result/summary/progress markers; failure before capability measurement was `ValueError("records exceed chunk capacity")`. This is infrastructure/pre-result evidence only, not scientific PASS/FAIL.
- seed `37641`: final untouched scientific seed from the original frozen `(17641,27641,37641)` set. It remains LOCKED / UNCONSUMED at this issue's creation.

Never rerun/retry/redispatch #758 or #783 and never reuse their trigger/result/checkpoint namespaces.

## Corrected repaired scientific surface

The future seed37641 runner must use the merged #785 bridge, not the raw #776 adapter materializer.

Frozen relevant identities:
- #785 bridge `tam_research/aera_issue785_eval_materialization_bridge_cpu.py` blob `aa90b3ae3266786323b0517de8385135869c3653`
- #750 corrected evaluator wrapper blob `afc939a69633f68ded05eb585c95a599a8f5c981`
- consumed #776 repaired scientific adapter blob `1489df753f2eee0f3acc5aba049c0eeb40fce10d`
- #748 frozen base harness blob `40003b68987d026265b1d53fcb637f18277f9cac`
- #746 capability gate blob `981602432b684989f7a5011ff953e6965368c5e5`
- integrated repaired event-memory model blob `b695b1b7b7476be3a16433c96dff32870f2e1f49`
- v18 core blob `97861a2407876f62665b13140c2135b4a11d4597`
- delta-memory blob `ec0b5d29b3d4ac27bd60fd9c152480b9f177c3e9`.

The bridge must remain the sole evaluation materialization source for the successor scientific surface and must resolve exactly the authoritative deterministic 432-case #750 materialization. Do not copy or alter the #750 retry algorithm.

## Frozen scientific experiment contract

Preserve the #746/#748 experiment contract without weakening or reinterpreting anything:
- trained variants `A_backbone`, `B_backbone_plus_memory`, `C_backbone_plus_routing`, `D_combined`;
- reference `E_simple_retrieval` and inherited recurrent control handling;
- learned A/B/C/D use `IntegratedEventMemoryCapabilityLM` at the original scientific `HardwareAERAConfig` (`d_model=128`, four stages, chunk 256, memory dim 32, etc.);
- exactly 432 held-out evaluation cases / 432 unique IDs;
- non-reset distance counts: 108 each at 2, 8, 32 chunks;
- reset distance counts: 54 each at 8, 32 chunks;
- optimizer steps 1024;
- tokens per variant 16,777,216;
- checkpoint every 2,097,152 tokens / 128 optimizer steps;
- maximum 3600 GPU seconds per variant/seed;
- AdamW lr `3e-4`, min lr `3e-5`, warmup 64, betas `(0.9,0.95)`, wd `0.1`, grad clip `1.0`;
- microbatch 2, grad accumulation 2, sequence chunks 16;
- evaluation batch 8, bootstrap 10,000;
- latency batch 8, warmup 3, timed 10;
- equal-token and equal-GPU-time comparisons both required.

Frozen thresholds remain exactly:
- long-distance `B-A >= 0.10`;
- paired CI95 lower bound `>= 0.05` under the already-frozen decision implementation;
- correction gain `>= 0.10`;
- stale-value error reduction `>= 0.10`;
- long-distance `B-C >= 0.05`;
- reset leakage `<= 0.01`;
- simple-retrieval dominance margin `0.05`;
- memory inference latency multiplier `<= 2.0`;
- memory train-time multiplier `<= 1.5`;
- parameter delta fraction `<= 0.05`.

No benchmark, model equation, memory transport, event addressing, optimizer, budget, threshold, seed policy, or decision rule may change in this successor.

## Fresh seed37641 namespace

Reserve only; do not consume at issue creation:
- preauth title: `[aera-event-memory-repair-seed37641-preauth-v1]`
- L4 title: `[aera-event-memory-repair-seed37641-l4-v1]`
- result: `/vol/aera-capability/event-memory-repair-v2-seed37641/result.json`
- checkpoints: `/vol/aera-capability/event-memory-repair-v2-seed37641/checkpoints`

The old seed17641 and seed27641 namespaces/prefixes may appear only as explicit consumed-history guards, never as executable destinations.

## Allowed implementation

Prefer additive successor files only, leaving all merged #746/#748/#750/#770/#776/#785 files unchanged:
1. one seed37641 scientific adapter that inherits/reuses the repaired #776 train/eval/decision surface but binds evaluation materialization through the merged #785 bridge and hard-guards scientific execution to exactly seed `37641` only when explicit runtime authorization is supplied;
2. one Modal runner with a zero-GPU preflight and one separately gated L4 scientific function; preflight must construct no model, use no scientific seed, write nothing, and prove the reserved result absent and checkpoint directory absent/empty;
3. one owner-only issue-trigger workflow keyed only to the exact fresh preauth/L4 titles, Actions attempt 1, exact-source binding, unique-trigger checks, successful preauth-evidence requirement, and separate explicit L4-authorization-comment requirement;
4. one protocol JSON;
5. one CPU/static contract test.

The runner may add heartbeat/progress logging and durable evidence recording only; it may not alter scientific training/evaluation/decision semantics.

## Required CPU/static gates

Fail closed unless all hold:
- every frozen source blob listed above is exact;
- #785 bridge materialization is deterministic and exactly 432 unique cases with frozen geometry;
- successor scientific adapter resolves `evaluation_cases` through #785/#750 and not raw #748 base materialization;
- scientific model construction is blocked unless seed is exactly `37641` and an explicit runtime authorization flag is true;
- seeds `17641` and `27641` are permanently rejected;
- all A/B/C/D scientific variants construct the repaired integrated class with frozen scientific dimensions;
- exactly one GPU declaration exists and only on the future scientific function;
- preflight code path contains no model construction/training/evaluation/result/checkpoint write;
- fresh prefixes/result/checkpoint paths match this issue exactly;
- old consumed scientific namespaces are not executable destinations;
- FT1-FT9 and #785 materialization tests remain green;
- no scientific seed is consumed by CPU tests.

## Process

Before every mutation, fresh-audit live `main`, newest AERA issues/PRs, and competing work.

Before any branch/ref move:
1. create candidate files as detached blobs on exact current main;
2. syntax/static validate without Modal/GPU/scientific execution;
3. create detached tree/commit;
4. verify exact additive diff and frozen inherited identities;
5. freeze source SHA/tree, candidate blobs/tree/commit, branch/path names, namespace, seed and authority ceiling in this issue;
6. only then create one immutable branch/PR;
7. consume exactly one original full-repository CPU CI attempt; never rerun/retry/redispatch it.

CPU-green reaches only a separate explicit merge boundary. A merged/green runner reaches only a separate canonical zero-GPU preauthorization trigger. A successful preauthorization still does NOT authorize L4: explicit user wording such as `Authorize repaired seed 37641 L4` is required later.

## Authority ceiling

ZERO Modal scientific execution now; ZERO GPU/L4/A100/H100 allocation; ZERO scientific model construction/training/evaluation; ZERO seed `37641` consumption; ZERO scientific checkpoint/result creation; ZERO retry of consumed seeds; ZERO systems-optimization authority; ZERO architecture freeze; ZERO S2/replication credit; ZERO 100M/scaling/full pretraining; ZERO breakthrough/AGI/SSI claim.

Even a future seed37641 PASS would be only the preregistered bounded capability evidence for this repaired architecture; higher-stage interpretation or scaling would require a separate evidence review and authorization.

**Comment by vinceackermann2-sys** [comment]
## #790 descendant implementation-base audit after unrelated #789

Fresh audit immediately before this record:
- preregistered scientific source remains `SOURCE_MAIN=5e5d746c70380eb4819534eb5a061a2d142fa26f`, `SOURCE_TREE=2348be0e21c107d254146a7426b432f6f808687f`;
- live implementation base is now `IMPLEMENTATION_BASE_MAIN=1c0ecd321a530daee32a4603a9cd6fb201be2d8a`, `IMPLEMENTATION_BASE_TREE=00dd4e489f5ad70fc68b5d195a9ba820ac299a54`;
- live main is a direct merge descendant of the preregistered source through unrelated CORTEX-S PR #789; the preregistered AERA source is its first parent;
- no newer competing AERA capability successor was found after #790;
- current live tree retains the #776 workflow blob `d4d5e3153102b6946c057e8deb257a68b57d3f2f` and the merged #785 bridge content matching frozen blob `aa90b3ae3266786323b0517de8385135869c3653`;
- all #790 scientific identities, seed policy, benchmark geometry, thresholds, and authority ceiling remain bound to the preregistration and are not reinterpreted by this implementation-base movement.

Any #790 detached implementation must use exact live parent `1c0ecd321a530daee32a4603a9cd6fb201be2d8a` / tree `00dd4e489f5ad70fc68b5d195a9ba820ac299a54` while preserving the preregistered scientific source identities. ZERO GPU/Modal scientific execution and seed `37641` remains locked/unconsumed.

**Comment by vinceackermann2-sys** [comment]
## #790 pre-implementation seed37641 runner freeze

Fresh audit immediately before this freeze:
- scientific source remains `5e5d746c70380eb4819534eb5a061a2d142fa26f` / tree `2348be0e21c107d254146a7426b432f6f808687f`;
- implementation content was authored against `1c0ecd321a530daee32a4603a9cd6fb201be2d8a` / tree `00dd4e489f5ad70fc68b5d195a9ba820ac299a54`;
- live `main` advanced afterward only through unrelated CORTEX-S PR #798 to `9f4196b10674c6eb0454dc2d3459a7c283f16637` / tree `3882d1d7aba003a13a36ef55960d3f3f59dec92e`;
- #798 contains no AERA paths; #790 remains the newest AERA successor found;
- the same five candidate blobs were transplanted unchanged onto exact live `main` before this freeze;
- compare `9f4196b...011a38e` is exactly one commit ahead, five added files, zero deletions, no inherited-file modifications.

SOURCE_MAIN=5e5d746c70380eb4819534eb5a061a2d142fa26f
SOURCE_TREE=2348be0e21c107d254146a7426b432f6f808687f
IMPLEMENTATION_BASE_MAIN=1c0ecd321a530daee32a4603a9cd6fb201be2d8a
IMPLEMENTATION_BASE_TREE=00dd4e489f5ad70fc68b5d195a9ba820ac299a54
REF_BASE_MAIN=9f4196b10674c6eb0454dc2d3459a7c283f16637
REF_BASE_TREE=3882d1d7aba003a13a36ef55960d3f3f59dec92e
BRANCH=infra/aera-issue790-seed37641-runner-v1
PREAUTH_PREFIX=[aera-event-memory-repair-seed37641-preauth-v1]
L4_PREFIX=[aera-event-memory-repair-seed37641-l4-v1]
RESULT_PATH=/vol/aera-capability/event-memory-repair-v2-seed37641/result.json
CHECKPOINT_DIR=/vol/aera-capability/event-memory-repair-v2-seed37641/checkpoints
MODEL_SEED=37641
ADAPTER_BLOB=931ba460cae55ceb940b61d92288668f6e02ff53
PROTOCOL_BLOB=9f865dff148aa46ffd9957259aeb4c0591595cad
RUNNER_BLOB=58bc3bbe16492fa8161456ea5103653dc320f6b6
WORKFLOW_BLOB=250f966c2c1ced919663ca8ee57aa03a2edf8852
RUNNER_TEST_BLOB=94670f0f77f5ee86ce58ac9e5b3af29be22f2b7d
FROZEN_TREE=ef3297c4890c2d0c68556a1482d8884f647c435a
FROZEN_COMMIT=011a38e149e0883a4883de0af919b7ce065c1d38

Frozen additive paths only:
1. `tam_research/aera_issue790_event_memory_seed37641_scientific_adapter.py`
2. `docs/aera_issue790_event_memory_seed37641_protocol.json`
3. `modal_aera_issue790_event_memory_seed37641_runner.py`
4. `.github/workflows/aera-issue790-event-memory-seed37641-runner.yml`
5. `tests/test_aera_issue790_event_memory_seed37641_runner_cpu.py`

The superseded detached tree `b1810aac43d82960a52dc48a7ce96c0660a9fdc4` / commit `960fb39f973c181f525d073f6b96769d77cb4406` were never referenced and are inert after live-main advancement.

An accidental connector-created ref `__tmp_should_not_create__` points exactly to live `main` `9f4196b10674c6eb0454dc2d3459a7c283f16637`; it contains zero #790 candidate changes, is not an implementation branch, is not a trigger/evidence namespace, and must never be used as #790 evidence. The available connector exposes no delete-ref mutation, so it remains explicitly inert.

Authority ceiling remains unchanged: ZERO Modal/scientific execution; ZERO GPU allocation; ZERO scientific model construction/training/evaluation; seed `37641` remains unconsumed; ZERO checkpoint/result creation; ZERO systems optimization, architecture freeze, S2/replication, scaling/100M/full pretraining, or breakthrough claim. Exactly one intended implementation branch/PR and one original full-repository CPU CI attempt are authorized next; that attempt must never be rerun/retried/redispatched.

**Comment by vinceackermann2-sys** [comment]
## #790 immutable CPU-green implementation evidence for PR #800

Fresh boundary audit immediately before this evidence record:
- live `main=9f4196b10674c6eb0454dc2d3459a7c283f16637`, tree `3882d1d7aba003a13a36ef55960d3f3f59dec92e`;
- PR #800 remains open, mergeable, not merged, exact frozen head `011a38e149e0883a4883de0af919b7ce065c1d38`;
- PR #800 remains exactly 5 changed files, `+1459/-0`;
- #790 remains the newest AERA successor found; no newer competing AERA successor exists.

Authoritative implementation freeze:
- freeze comment: `5600299057`
- frozen head: `011a38e149e0883a4883de0af919b7ce065c1d38`
- frozen tree: `ef3297c4890c2d0c68556a1482d8884f647c435a`
- branch: `infra/aera-issue790-seed37641-runner-v1`
- PR: #800

Sole original full-repository CPU CI attempt for this frozen head:
- workflow: `CI`
- run: `34340049838`
- job: `102428456557`
- attempt: `1`
- PR test merge commit: `bb7d1b00610b803325d8bb762818e173c83b2fdb`
- tested base: `9f4196b10674c6eb0454dc2d3459a7c283f16637`
- tested head: `011a38e149e0883a4883de0af919b7ce065c1d38`
- conclusion: **SUCCESS**
- exact pytest result: **`1246 passed, 15 warnings in 170.18s (0:02:50)`**
- syntax-check/setup steps: success

This Actions attempt is consumed and immutable. NEVER rerun/retry/redispatch run `34340049838` or job `102428456557`.

Evidence classification: CPU/static implementation PASS only. It proves the frozen #790 additive infrastructure integrates cleanly with the exact tested base and full repository CPU suite. It is not scientific capability evidence.

Authority after this PASS remains: ZERO Modal/scientific execution; ZERO GPU allocation; ZERO scientific model construction/training/evaluation; seed `37641` remains LOCKED / UNCONSUMED; ZERO checkpoint/result creation; ZERO systems optimization, architecture freeze, S2/replication, scaling/100M/full pretraining, or breakthrough claim.

This CPU-green state reaches only the separate explicit merge boundary. Merge #800 requires a fresh exact-state audit plus explicit user authorization `Authorize merge #800`. A future merge would still authorize only the separately gated zero-GPU preauthorization stage, not L4/scientific execution.

**Comment by vinceackermann2-sys** [comment]
## #790 repaired seed 37641 L4 scientific authorization v2

Explicit user authorization received: `Authorize seed37641 L4 v2`.

Authorize main: `3d9716d1f786f2b821d2dcbb68441ac490358ecf`
Authorize seed: `37641`
Preauth evidence issue: `#836`
Preauth evidence comment: `5616487624`

This authorizes exactly one canonical `[aera-event-memory-repair-seed37641-l4-v2]` scientific trigger bound to the authorized main and seed above, and exactly its original workflow attempt 1. Never rerun/retry/redispatch. It does not authorize any additional seed, replication, architecture freeze, S2, scaling/full pretraining, or breakthrough claim. The scientific result/checkpoint namespaces remain exactly those frozen for seed 37641.
